### ISSUE TITLE
Credentials in submitted code

### ISSUE THREAD

**Issue by jtacoma** [ElwynPalmerton/Zybriqs#1]
https://github.com/ElwynPalmerton/Zybriqs/blob/2432f7514cc6060fed60a5c1310fd705014ffaa7/app.js#L50

This line includes credentials for a Mongo database. I haven't tried them to verify they work, but it might be the same database used in production? If so, it could be a problem for people who sign up with the same username and password that they use elsewhere.

**Comment by jtacoma** [comment]
Ideally the credentials would be kept outside of the code, in (say) a service that manages secrets.

A faster way to limit the damage here is to add a message during sign up that warns people to use a throwaway password. As long as everyone knows it's experimental, this could be enough until the credentials are removed from the code. They'd still be visible in code history though, so (if security matters here) you'll want to create a new database with new credentials that haven't been made public before.

**Comment by ElwynPalmerton** [comment]
Ok, thanks for the warning. I should not have done that. I will work on
this this week and make sure that I can fix everything - including hiding
the Mongo credentials and adding a warning to not use an important
password. Aside from passwords and emails the site does not take or store
any personal information. Nonetheless, I would like to make this as secure
as possible. Largely, this was a project just for me to practice doing all
of this stuff and I still have much room to improve.

Thank you,
Elwyn Palmerton

**Comment by jtacoma** [comment]
I suppose you're right: it's not a high severity problem. Anyway, it's definitely my mistake to have posted a public issue on GitHub about a security vulnerability! I should report this kind of problem more discreetly so it can be fixed before potential attackers find out about it. Feel free to delete/hide this issue if you see any admin functions to do so.

**Comment by ElwynPalmerton** [comment]
No worries, like I said, I appreciate the heads-up. I just got this online
a couple weeks ago and was considering it a work-in-progress. Gonna get
back on this tomorrow. It needs many other tweaks as well.

Anyway, I hope you don't mind me asking how you found this project? I have
been teaching myself web development for about a year now and this is the
first "big" (full-stack) project that I've made. I think that it's a
good gauge of where I'm at right now so if you have any feedback it would
be greatly appreciated.

Eventually, I would like to find work as a web developer but right now I am
just trying to learn and make fun projects. As I said, I'm totally
self-taught so any advice or guidance would be super helpful.

Best regards,
Elwyn
